src/pal/inc/rt/sal.h

```diff
@@ -127,7 +127,7 @@
 
    The success criteria can be specified with the _Success_(expr) annotation:
      _Success_(return != FALSE) BOOL
-     PathCanonicalizeA(_Out_writes_(MAX_PATH) LPSTR pszBuf, LPCSTR pszPath) :
+     PathCanonicalizeA(_Out_writes_(MAX_LONGPATH) LPSTR pszBuf, LPCSTR pszPath) :
```

Comment:
PathCanonicalize is defined is hard-coded to use MAX_PATH sized buffer (see https://msdn.microsoft.com/en-us/library/windows/desktop/bb773569(v=vs.85).aspx). It is confusing to change the example to different size.